packages/cli/src/commands/build.ts

```diff
@@ -304,7 +304,12 @@ async function doBuild(
     ...pickOverrides(localConfig),
   };
 
-  await setMonorepoDefaultSettings(cwd, workPath, projectSettings, output);
+  if (
+    projectSettings.rootDirectory !== null &&
+    projectSettings.rootDirectory !== '.'
+  ) {
+    await setMonorepoDefaultSettings(cwd, workPath, projectSettings, output);
+  }
```

Comment:
Why exactly do we need to skip in this case?

Reply:
We thought about it like in the case of `vercel/vercel`. There is no way to really know what the user wants to output and we just should rely on whatever their build script is in this instance. As for example, we use turbo build, which then eventually calls `packages/cli/scripts/build.ts` which outputs the output of cli build into the root `public` directory.